src/libs/antares/study/area/area.cpp

```diff
@@ -53,6 +53,7 @@ void Area::internalInitialize()
 Area::Area() :
  index((uint)(-1)),
  enabled(true),
+ adequacyPatchMode(Data::adqmPhysicalAreaOutsideAdqPatch),
```

Comment:
I suggest in-class initialization. Also,
```suggestion
 adequacyPatchMode(Data::AdequacyPatch::PhysicalAreaOutside),
```